src/pages/workspaces/workspace/workflows/WorkflowView.js

```diff
@@ -395,11 +395,21 @@ const WorkflowView = _.flow(
     // savedConfig: unmodified copy of config for checking for unsaved edits
     // modifiedConfig: active data, potentially unsaved
     const {
-      isFreshData, savedConfig, entityMetadata, launching, activeTab, useCallCache, deleteIntermediateOutputFiles,
+      currentSnapRedacted, isFreshData, savedConfig, entityMetadata, launching, activeTab, useCallCache, deleteIntermediateOutputFiles,
       entitySelectionModel, variableSelected, modifiedConfig, updatingConfig
     } = this.state
     const { namespace, name, workspace } = this.props
     const workspaceId = { namespace, name }
+    let workflowMetadata
```

Comment:
I suspect this can go away if this is inlined after deleting the unused code below, but just as an FYI:

We avoid using `let` unnecessarily because it effectively creates mutable state, which can make code harder to reason about. Cases like this can be expressed like:

```js
const maybeValue = computeTheValue ? { x: 1 } : undefined
```

If the computation is more complex, it could also be split out as a locally-scoped helper function, which could then be called conditionally.

By being strict about this, we can treat the (necessary) `let` as a strong signal to readers that says: 'careful, there is state here'